coss/opensource_clubs/templates/opensource_clubs/activities_page.html

```diff
@@ -0,0 +1,31 @@
+{% extends "home/about_page.html" %}
```

Comment:
Why is this needed?

Reply:
Same as the FAQ page, this inherits from the about_page template. We should probably refactor this to have a more generic name, since we use it on various other occasions besides the About.